Client/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -9674,3 +9674,36 @@ CClientSearchLight* CStaticFunctionDefinitions::CreateSearchLight(CResource& Res
 
     return nullptr;
 }
+
+bool CStaticFunctionDefinitions::ResetSurfaceInfo()
+{
+    CSurfaceType* pSurfaceInfo = m_pClientGame->GetSurfaceInfo();
+    DWORD dwSurfaceInfo = (DWORD)pSurfaceInfo;
+
+    CSurfaceType pOriginalSurfaceInfo = m_pClientGame->GetOriginalSurfaceInfo();
+    DWORD dwOriginalSurfaceInfo = (DWORD)(&pOriginalSurfaceInfo);
+
+    short sOffset = 144;

```

Comment:
If this is the offset of `CSurfaceType::surfType` then use the **offsetof** macro from **stddef.h**:
`offsetof(CSurfaceType, surfType)`